.github/workflows/diff.yml

```diff
@@ -1,13 +1,40 @@
 name: 'Create ROBOT diffs on Pull requests'
 
-on: 
+on:
```

Comment:
Shouldn't this workflow run after every commit made to a PR? I'm making the assumption that every open branch is associated with a PR. So with that in mind, shouldn't there be
```
on:
  pull_request:
    branches: [ main ]
```

Reply:
Ah no, this action fires when a specific command is spoken in a comment on the PR, like "kgcl" bot.